### PR TITLE
style: fix transfer search padding in rtl

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix Transfer search padding in RTL          |
| 🇨🇳 Chinese | 修复 Transfer 查询框在 RTL 模式下的 `padding`           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/transfer/style/index.less [hunk 1]
```diff
@@ -35,7 +35,7 @@
     }
 
     &-search {
-      padding-right: 24px !important;
```
Comment (afc163) [inline components/transfer/style/index.less:38]: 看看当初为啥加这个 `!important`，避免改挂。

Comment (yoyo837) [reply components/transfer/style/index.less:38]: #22126 @AshoneA

Comment (xrkffgg) [reply components/transfer/style/index.less:38]: 没发现有啥问题

Comment (AshoneA) [reply components/transfer/style/index.less:38]: 有点记不起来了，不过试了下应该可以去掉。

**Review by AshoneA — APPROVED** [review]

**Pull request merged**